Title: Human gut metagenomics : five years of exciting exploration
Date: 2014-07-07 16:03:00
Tags: MetaHIT, HMP, Metagenomics, microbiota
Slug: 2014-07-07-gut-metagenomics-five-years.html

<img itemprop="image" src="http://eldermet.ucc.ie/wp-content/uploads/IHMC-logo.jpg" alt="IHMC logo" style="float: left; border: 20px solid white; width: 150px;"/>

I defended my [PhD](http://julientap.free.fr/These_Julien_Tap.htm "Julien Tap These") in December 2009 and I knew that gut metagenomics will become more and more exiting with such big consortia such as [MetaHIT](http://www.metahit.eu/ "metahit website") and [HMP](http://www.hmpdacc.org/). Combining with other gut microbiome initiatives within the [international human microbiome consortium](http://www.human-microbiome.org/), they contributed to establish new resources, methods and clinical potential applications using metagenomics datasets.

I made a very rapid review here of huge step articles published during the last five years (I probably forget some of them).

### New gut metagenomics resources ###
The first non redundant human gut microbial gene catalog was released in 2010. It contains 3.3 millions genes obtained from 124 individuals [^1]. This study confirmed the existence of a phylogenetic core of microbial species and a functional genes core which was observed respectively by my lab thesis[^2] and by Turnbaugh and colleagues[^3]. {{more}}

After the two important HMP resources articles with reference genomes from the human microbiome[^4] and  the diversity of all body site of the human microbiome[^5], the MetaHIT consortium released in 2014 an updated version of the genes catalog including more than 1,200 samples from three continents reaching 10 milions non redundant genes[^6].

### New gut metagenomics methods ###
Although generating human gut gene catalog is already a huge step of methodological development, other methods have been tested to untangle the complexity of metagenomics dataset. First, unsupervised stratification allowed to discover that there is a limited number of community types called enterotypes[^7] which could have a long term link with diet[^8]. this discovery was subjected to some debate but a new study[^9] which used the bayesian inference and multinomial dirichlet developed by Holmes and collegues[^10] confirmed the original observation. 

Using [universal markers genes](http://julientap.com/post/2013-10-30-new-metagenomic-marker-genes-method-microbiota.html) method to study the human gut microbiota, scientists captured in silico twice as species and a practical example has allowed to describe that species richness was lower in ulcerative colitis patients compared to healthy individuals [^11]. [Another method](http://http://julientap.com/post/2012-12-14-genomic-variation-landscape-of-the-human-gut-microbiome.html) was also developed to call single nucleotide polymorphism from metagenomics data[^12]. Each individual have its own microbial stable genotype : while species abundance could vary, it appears that each individual have their own genotype by species.

The last method published in Nature Biotechnology[^13] allowed to reconstruct species genomes from metagenomics data. Using co-abundance across individual, this new methodology permitted to identify and to assembly of genomes and genetic elements in complex metagenomic samples without using reference genomes.

### New gut metagenomics clinical potential applications ###
All this resources and new methods was and will be used for further clinical metagenomics associations. Interestingly, the simplest index, the gut microbial genes richness, was correlated with metabolic markers[^14] and was prognostic to weight loss during low caloric diet over time[^15]. Another study from the [ELDERMET consortium](http://eldermet.ucc.ie/publications/) showed that gut microbiota from elderly patient is associated with health care management and diet [^16]. The HMP consortium provided a huge collection of articles in [PLoS journal](http://www.ploscollections.org/hmp "PLoS HMP"). 

---
I'm wondering with such huge improvement in term of next generation sequencing like the [MiniION Oxford Nanopore](https://www.nanoporetech.com/technology/the-minion-device-a-miniaturised-sensing-system/the-minion-device-a-miniaturised-sensing-system) how those discoveries will find a clinical translation.


### References ###
[^1]: J Qin et al. MetaHIT consortium. [A human gut microbial gene catalogue established by metagenomic sequencing](http://www.nature.com/nature/journal/v464/n7285/abs/nature08821.html). 2010. Nature 464 (7285), 59-65

[^2]: J Tap et al. 	[Towards the human intestinal microbiota phylogenetic core](http://onlinelibrary.wiley.com/doi/10.1111/j.1462-2920.2009.01982.x/abstract). 2009. Environmental microbiology 11 (10), 2574-2584 

[^3]: PJ Turnbaugh et al. [A core gut microbiome in obese and lean twins](http://www.nature.com/nature/journal/v457/n7228/full/nature07540.html). 2009. Nature

[^4]: Nelson KE et al. [A catalog of reference genomes from the human microbiome](http://www.ncbi.nlm.nih.gov/pubmed/20489017). Science 2010 May 21

[^5]: HMP consortium. [Structure, function and diversity of the healthy human microbiome.](http://www.ncbi.nlm.nih.gov/pubmed/22699609) Nature 2012 Jun 14.

[^6]: Li et al. MetaHIT consortium. [An integrated catalog of reference genes in the human gut microbiome.](http://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.2942.html) Nature Biotechnology. 2014

[^7]: M Arumugam, J Raes et al.[ MetaHIT consortium. Enterotypes of the human gut microbiome](http://www.nature.com/nature/journal/v473/n7346/abs/nature09944.html). 2011. Nature 473 (7346), 174-180

[^8]: Wu GD et al. [Linking long-term dietary patterns with gut microbial enterotypes](http://www.ncbi.nlm.nih.gov/pubmed/21885731). Science 2011 Oct 7

[^9]: Ding T and Schloss PD. [Dynamics and associations of microbial community types across the human body.](http://dx.doi.org/10.1038/nature13178) Nature. 2014 May 15;509(7500):357-60

[^10]: Holmes et al. [Dirichlet Multinomial Mixtures: Generative Models for Microbial Metagenomics.](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0030126) PLoS One. 2012. 10.1371/journal.pone.0030126

[^11]: Sunagawa et al. [Metagenomic species profiling using universal phylogenetic marker genes.](http://dx.doi.org/10.1038/NMETH.2693) Nature Methods.2013

[^12]: Schloissnig et al. [Genomic variation landscape of the human gut microbiome.](http://www.nature.com/nature/journal/vaop/ncurrent/full/nature11711.html). 2013. Nature 493, 45–50.

[^13]:Nielsen, Almeida et al. MetaHIT consortium. [Identification and assembly of genomes and genetic elements in complex metagenomic samples without using reference genomes.](http://www.nature.com/nbt/journal/vaop/ncurrent/full/nbt.2939.html) Nature Biotechnology. doi:10.1038/nbt.2939

[^14]: Le Chatelier et al. MetaHIT consortium. [Richness of human gut microbiome correlates with metabolic markers.](http://www.nature.com/nature/journal/v500/n7464/abs/nature12506.html) 2013. Nature. Nature 500 (7464), 541-546

[^15]: Cottilard et al. [Dietary intervention impact on gut microbial gene richness.](http://www.nature.com/nature/journal/v500/n7464/full/nature12480.html) Nature; 2013. 500, 585–588.

[^16]: Claesson MJ et al.  [Gut microbiota composition correlates with diet and health in the elderly.](http://www.ncbi.nlm.nih.gov/pubmed/22797518) Nature. 2012 Aug 9;488(7410):178-84. doi: 10.1038/nature11319.

